internal/services/cragcast/types.go

```diff
@@ -0,0 +1,34 @@
+package cragcast
+
+import "time"
+
+type Cragcast struct {
+	Days []*DailyForecast
+}
+
+type DailyForecast struct {
+	Date time.Time
+	// these are the headlines for the day
+	TotalPrecipMm       *float32
+	ChanceOfPrecip      *float32
+	LowOfDayFahrenheit  *float32
+	HighOfDayFahrenheit *float32
+	avgSkyCoverPercent  *float32
```

Comment:
Did you mean to make some of these [private (unexported)](https://go.dev/ref/spec#Exported_identifiers)? Implications of this would be:

* Field is inaccessible from importing packages.
* Field will not serialize when using `json.Marshal`.

Reply:
woops, thanks